kep/119-node-resource-topology-aware-scheduling/README.md

```diff
@@ -185,6 +187,22 @@ The algorithm which implements SingleNumaNode policy is following:
 	return nil
 }
 ```
+
+### Description of the Score Algorithm
+
+The algorithm which prioritize based on specific scoring strategy is as follows:
+```go
+    numaScores := make([]int64, len(numaList))
+    for _, numa := range numaList {
+    	// score is a pointer to a function that can be implemented in various ways 
+        numaScores[numa.NUMAID] = score(requested, numa.Resources, resourceToWeightMap)
+    }
+
+    minScore := findMinScore(numaScores)
```

Comment:
why we use the min of numa scores as the plugin score?

Reply:
We return the minimal score to handle worst-case scenarios, For example:
Worker-Node 0:
NUMA 0 score: 50
NUMA 1 score: 70

Worker-Node 1:
NUMA 0 score: 30
NUMA 1 score: 90

Since we can't explicitly tell Kubelet which NUMA (NUMA 0 or NUMA 1) to select, it's better to assume Kubelet will choose the less-good NUMA, i.e. NUMA with the minimal score.
So we preferred to have a Worker-Node 0 and Worker-Node 1 with a final score of 50 and 30 respectively because it will **guarantee** to have the workloads deployed on a NUMA with a score of at least 50.

Reply:
> Since we can't explicitly tell Kubelet which NUMA (NUMA 0 or NUMA 1) to select, it's better to assume Kubelet will choose the less-good NUMA, i.e. NUMA with the minimal score.


Thanks. It's make sence.
It's better to add your explanation in this doc.

Reply:
> It's better to add your explanation in this doc.

Done.